keyboards/kprepublic/jj50v2/info.json

```
@@ -0,0 +1,81 @@
+{
+    "keyboard_name": "JJ50v2",
+    "manufacturer": "KPrepublic",
+    "url": "",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0x4B50",
+        "pid": "0x0050",
+        "device_version": "3.0.0"
+    },
```

migrated from `rules.mk`
```suggestion
    },
    "processor": "atmega32u4",
    "bootloader": "atmel-dfu",
    "features": {
        "bootmagic": true,
        "mousekey": true,
        "extrakey": true,
        "backlight": true,
        "rgblight": true
    },
```